gui/wxpython/gui_core/pyedit.py

```diff
@@ -269,6 +269,14 @@ def __init__(self, panel, guiparent, giface):
         self.overwrite = False
         self.parameters = None
 
+        # Get first (File) menu
+        menu = guiparent.menubar.GetMenu(0)
+        self.recent_files = RecentFilesMenu(
+            app_name='pyedit', parent_menu=menu, pos=1,
```

Comment:
Similarly to GetMenu(0), please document also the intention for pos=1.

Reply:
All right, I added it.